src/Caching/SqlServer/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- These projects depend on code which is not open source. -->
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <NoWarn>$(NoWarn);PKG0001</NoWarn>
```

Comment:
nit: Why are these settings here instead of in the src/Caching/SqlServer/src/Microsoft.Extensions.Caching.SqlServer.csproj file❔

Reply:
No particular reason, I just didn't bother changing it when I migrated them from extensions. I'll update them.